api/src/wfl/api/spec.clj

```diff
@@ -97,8 +100,16 @@
                                        ::unmapped_bam_suffix
                                        ::updated
                                        ::uuid]
-                              :req-un [::id
-                                       ::input_cram]))
+                              :req-un [::input_cram
+                                       ::sample_name]))
```

Comment:
`sample_name` is already listed under the optional workflow inputs (the module uses the name of the input_cram as the default if it is not specified)

Reply:
thanks very much

Reply:
Oh I didn't realize this is the schema for the workflows returned by WFL. The default inputs get set here: https://github.com/broadinstitute/wfl/blob/2fb87663e33649ecf84e7b3b9672691e228bce64/api/src/wfl/module/wgs.clj#L103

Reply:
Thanks. I'll just leave the wgs as requiring the cram